README.md

```diff
@@ -100,13 +109,23 @@ Stripe Payment Method other than "card" on the admin interface, you must include
 
 `app/views/spree/admin/payments/source_forms/existing_payment/stripe/`
 
-### Custom webhooks
+### Customizing Webhooks
+
+Solidus Stripe comes with support for a few [webhook events](https://stripe.com/docs/webhooks), to which there's a default handler. You can customize the behavior of those handlers by or add to their behavior by replacing or adding subscribers in the internal Solidus event bus.
+
+Each event will have the original Stripe name, prefixed by `stripe.`. For example, the `payment_intent.succeeded` event will be published as `stripe.payment_intent.succeeded`.
 
-You can also use [Stripe webhooks](https://stripe.com/docs/webhooks) to trigger
-custom actions in your application.
+Here's the list of events that are supported by default:
 
-First, you need to register the event you want to listen to, both [in
-Stripe](https://stripe.com/docs/webhooks/go-live) and in your application:
+        payment_intent.succeeded
+        payment_intent.payment_failed
+        payment_intent.canceled
+        charge.refunded
```

Comment:
Not against it, as I agree its clearer, but I'm afraid it'll be too easy to forget to add new events here. Your call.

Reply:
Yes, I guess it's the perennial problem of documentation getting out of sync… I added a note to the list of _core events_ in the Event class.